examples/ui/client-capabilities/interaction/text-area.js

```diff
@@ -0,0 +1,8 @@
+import { CodeLabel, CodeSection } from '../../documentationBase';
+import { registerComponents } from '../../rapidImports';
+
+registerComponents();
+
+export default function TextAreaDemo({ children, color }) {
+	return (<CodeSection><rapid-text-area autocapitalize="sentences" placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua">Text Area</rapid-text-area></CodeSection >)
```

Comment:
Can we get more examples here?

Min length, max length, rows, cols, pre filled value, disabled etc